src/add-to-app/android/plugin-setup.md

```diff
@@ -148,7 +148,7 @@ by using the same major semantic version in your
 existing app and Flutter module plugin.
 
 
-[ExoPlayer from the video_player plugin]: {{site.repo.plugins}}/blob/master/packages/video_player/video_player/android/build.gradle
+[ExoPlayer from the video_player plugin]: {{site.repo.plugins}}/blob/main/packages/video_player/video_player/example/android/build.gradle
```

Comment:
Nice catch! However looks we use this link as a reference for the `build.gradle` that adds exoplayer.

I think it might be this one?

```suggestion
[ExoPlayer from the video_player plugin]: {{site.repo.plugins}}/blob/main/packages/video_player/video_player_android/android/build.gradle
```